### ISSUE TITLE
mod isnt available for 1.20.1

### ISSUE THREAD

**Issue by owleague** [luavixen/damagenumbers#4]
as the title says and after launching the game with the mod this comes up
- Replace mod 'Damage Numbers' (damagenumbers) 1.0.0-fabric with any version that is compatible with:
- minecraft 1.20.1

**Comment by RashidFTW** [comment]
I had to edit the mod's manifest version to get it to work, I tested on 1.20.1. 
On the mod's modrinth page it says that it should support Fabric 1.19–1.20.4, so not sure why the depends version is set wrong in the manifest. 

As a workaround until @luavixen fixes the issue: 
1. Change the extension of the .jar file to .zip and open/extract it with any archiving tool such as: 7zip, WinRAR
2. Edit the fabric.mod.json file, change this line: `"minecraft": "~1.20.4",` to `"minecraft": ">=1.19 <=1.20.4",`
3. Save the file and repack the archive
4. Change back the extension from .zip to .jar
